v7/localsearch/conf.py.sample

```diff
@@ -28,17 +32,22 @@ BODY_END = """
   </div>
 </div>
 <script>
+"""
+BODY_END += """
```

Comment:
Do it in one assignment, using string concatenation instead. Also, use `json.dumps()` to escape the URL properly (adds quotes).